site/html/gallery.html

```diff
@@ -81,128 +132,302 @@
-      <i class="fa fa-arrow-up"></i>
-    </button>
-
-    <!-- Full screen modal -->
-   <div id = "myModal" >
-      <button type="button" id = "close-btn" aria-label="Close">X</button>
-      <div class="img-container">
-        <img class="modal-content" id="modal-img">
-      </div>
-   </div>
-
-
-   <div id="gallery">
-    <section class="sec-container" id="BRG_linux">
-      <div class="heading">
-        <h1>Images from Open Source and Linux</h1>
-      </div>
-
-      <div class="box">
-        <div class="img-col">
-          <img class = "view-image"src="/site/images/gallery/BRG_linux/photo1675142199.jpeg" alt="">
-          <img class = "view-image"src="/site/images/gallery/BRG_linux/photo1675509551 (1).jpeg" alt="">
-          <img class = "view-image"src="/site/images/gallery/BRG_linux/photo1675567713.jpeg" alt="">
-          <img class = "view-image"src="/site/images/gallery/BRG_linux/photo1675509551.jpeg" alt="">
-        </div>
-
-        <div class="img-col">
-          <img class = "view-image"src="/site/images/gallery/BRG_linux/photo1675509600.jpeg" alt="">
-          <img class = "view-image"src="/site/images/gallery/BRG_linux/photo1675526474.jpeg" alt="">
-          <img class = "view-image"src="/site/images/gallery/BRG_linux/photo1675567714 (1).jpeg" alt="">
-          <img class = "view-image"src="/site/images/gallery/BRG_linux/photo1675509551 (2).jpeg" alt="">
+      <!-- Back to top button -->
+      <button
+        type="button"
+        class="btn btn-success btn-floating btn-lg"
+        id="btn-back-to-top"
+      >
+        <i class="fa fa-arrow-up"></i>
+      </button>
+
+      <!-- Full screen modal -->
+      
+      <div id="myModal">
+        <div class="top">
+          <i class="fa-solid fa-xmark" id="close-btn"></i>
         </div>
-
-        <div class="img-col">
-          <img class = "view-image"src="/site/images/gallery/BRG_linux/photo1675567715.jpeg" alt="">
-          <img class = "view-image"src="/site/images/gallery/BRG_linux/photo1675567714.jpeg" alt="">
-          <img class = "view-image"src="/site/images/gallery/BRG_linux/photo1675509600 (2).jpeg" alt="">
-          <img class = "view-image"src="/site/images/gallery/BRG_linux/photo1675567713 (1).jpeg" alt="">
+        <div class="img-container">
+          <div class="left center">
+            <i class="fa-solid fa-angle-left userInter" id="previousButton"></i>
+          </div>
+          <div class="imageBox">
+            <img class="modal-content" id="modal-img" />
+          </div>
+          <div class="right center">
+            <i class="fa-solid fa-angle-right userInter" id="nextButton"></i>
+          </div>
         </div>
       </div>
-    </section>
+      
+      </div>
+      <div id="gallery">
+        <section class="sec-container" id="BRG_linux">
+          <div class="heading">
+            <h1>Images from Open Source and Linux</h1>
```

Comment:
Could you please set it to `Images from Open Source and LInux Workshop`

Reply:
I don't get it can you describe it more clearly.

Reply:
I just asked if you could change the `h1` text to "Images from Open Source and Linux Workshop".